runtime/rastrace/method_trigger.c

```diff
@@ -547,6 +546,43 @@ decimalString2Int(J9PortLibrary* portLibrary, const char *decString, I_32 signed
 	return num;
 }
 
+/**************************************************************************
+ * name        - setMaxStringLength
+ * description - Set max method string argument length
+ * parameters  - thr, trace options, atRuntime flag
+ * returns     - JNI return code
```

Comment:
Please correct this documetation:
* it isn't just about arguments; string return values are traced as well
* there's no parameter named 'thr'